DI-2497/run_describe_beds_old_panelapp_versions.sh

```
@@ -0,0 +1,57 @@
+then
+  echo "Error: \"{$PROJECT_WITH_TEST_CODE}\"-\"${PROJECT_NAME_WITH_TEST_CODE}\" is not a valid 002 project ."
+  exit 1
+fi
+
+if [[ $(grep -ic -E "CEN|TWE" <<< "${PROJECT_NAME_WITH_TEST_CODE}") -eq 0 ]]
+then
+  echo "Error: \"{$PROJECT_WITH_TEST_CODE}\"-\"${PROJECT_NAME_WITH_TEST_CODE}\" is not a valid 002 dias project ."
+  exit 1
+fi
+
+for ID in ${panel_beds_R134[@]};do 
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)